en/docs/key-concepts/identity-provisioning-and-its-standards.md

```diff
@@ -27,14 +25,11 @@ process.
 
 ![Inbound provisioning using WSO2 Identity Server](../../assets/img/concepts/inbound-provisioning.png)
```

Comment:
the image paths should start with "/assets/img..." instead of "../../assets/img"

Reply:
please fix in all places